app/javascript/components/analytics/widgets/EnrollerStatistics.js

```diff
@@ -15,38 +15,22 @@ class EnrollerStatistics extends React.Component {
           <Card.Body className="mt-4">
             <Row id="monitoreeAnalytics" className="mx-4 mt-3 g-border-bottom-2">
               <Col>
-                <Row>
-                  <div className="h4">TOTAL MONITOREES</div>
-                </Row>
-                <Row>
-                  <div className="display-3 h3">{this.props.total_monitorees}</div>
-                </Row>
+                <div className="h4 text-left">TOTAL MONITOREES</div>
```

Comment:
One small thing, we've had requests to actually use the header tags instead of the class `h3`.  Can you update this?